core/controllers/pages.py

```diff
@@ -18,11 +18,13 @@
 import urlparse
 
 from core.controllers import base
-from core.controllers import editor
 from core.domain import config_domain
 import feconf
 
 
+MODERATOR_REQUEST_FORUM_URL_DEFAULT_VALUE = (
```

Comment:
Good point. In that case we might as well leave it here for now. It's nice to have transparency about the featuring process for those who really care.